Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,42 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="installed_python",category=['miscellaneous'],prompt="I installed Python.",pool=True))
+
+label installed_python:
+    m 1sub "You did?"
+    m 1hub "That's great, [player]!"
+    m 1eub "What did you download it for?{nw}"
+    $ _history_list.pop()
+    menu:
+        m 1eub "What did you download it for?{fast}"
+        "A personal project.":
+            m 1hub "How fun!"
+            m 3hub "Good luck with whatever you're programming!"
+            m 3ekbfa "Maybe you could help me out with a few things?"
+            m 1eub "I'll teach you as much as I can!"
+            m 1hub "Maybe you could even teach me a few things!"
+            m 1eub "Ehehehe!"
+            m 3hua "Just ask me if you ever need help with anything!"
+    
+    "To help you come closer to our reality.":
```

Comment:
All menu options should be on the same indent level

Reply:
oop okay

Reply:
Fixed!